development/mimir-microservices-mode/docker-compose.jsonnet

```diff
@@ -294,7 +294,7 @@ std.manifestYamlDoc({
 
   memcached:: {
     memcached: {
-      image: 'memcached:1.6.19-alpine',
+      image: 'memcached:1.6.28-alpine',
```

Comment:
Unrelated but I needed to be able to `watch deletions` which was only available in a newer version and this matches what we run in production.